background/background.js

```diff
@@ -147,12 +136,22 @@ prefs.initializing.then(() => {
   refreshAllIcons();
 });
 
-navigatorUtil.onUrlChange(({tabId, frameId, transitionQualifiers}, type) => {
-  if (type === 'committed' && !frameId) {
+navigatorUtil.onUrlChange(({tabId, frameId, transitionQualifiers, url}, type) => {
+  if (type !== 'committed') {
+    msg.sendTab(tabId, {method: 'urlChanged'}, {frameId})
+      .catch(msg.ignoreError);
+  } else if (!frameId) {
+    if (url.includes('.user.') &&
+        /^(https?|file|ftps?):/.test(url) &&
+        /\.user\.(css|styl)$/.test(url.split(/[#?]/, 1)[0]) &&
```

Comment:
We need a `URLS.isUsercssSource` function

Reply:
Do we? Currently this check is performed only in one place so adding a function to `URLS` that's present in all other pages may be an overkill.

Reply:
Exposed it in usercssHelper.